header-footer-grid/templates/header-wrapper.php

```diff
@@ -11,7 +11,12 @@
 namespace HFG;
 
 use HFG\Core\Builder\Header as HeaderBuilder;
+
+$transparent_header_class = '';
```

Comment:
@preda-bogdan let's use a filter for the header classes, something like, hfg_header_classes where we hook this logic, in neve pro